website/docs/MigrationV12.md

```diff
@@ -0,0 +1,28 @@
+---
+id: migration-v12
+title: Migration to 12.0
+---
+
+Migration to React Native Testing Library version 12 from version 11.x should be a relatively easy task due small amount of breaking changes.
```

Comment:
```suggestion
React Native Testing Library 12 introduces a handful of breaking changes compared to 11.x. We believe they were necessary to improve the experience using the library and help the users fall into the pit of success when writing meaningful tests. You will find migration instructions for each and every change below.
```

Reply:
"help users fall into the pit of success when ..." 😂

Reply:
@thymikee Is this the pit you are referring to: https://blog.codinghorror.com/falling-into-the-pit-of-success/

Reply:
c'mon